packages/mail/src/mail.spec.js

```diff
@@ -17,7 +17,7 @@ before(() => {
  * Default mock header
  */
 beforeEach(() => {
-  sgClient.setDefaultHeader('X-Mock', 200);
+  sgClient.setDefaultHeader('X-Mock', 202);
```

Comment:
it was failing because of 200 and as mentioned in their v3 documentation, the status code for the success response is 202. I called the api and cross checked it as well.
v2 used to return 200 on success response ([link](https://docs.sendgrid.com/v2-api/mail)) and now in v3 it's changed to 202(mentioned at the last of the article).([link](https://docs.sendgrid.com/api-reference/mail-send/mail-send))